stream_alert/alert_processor/outputs.py

```diff
@@ -803,3 +805,272 @@ def dispatch(self, **kwargs):
+        self.base_url = None
+        self.auth_cookie = None
+
+    def get_user_defined_properties(self):
+        """Get properties that must be asssigned by the user when configuring a new Jira
+        output.  This should be sensitive or unique information for this use-case that needs
+        to come from the user.
+
+        Every output should return a dict that contains a 'descriptor' with a description of the
+        integration being configured.
+
+        Jira requires a username, password, URL, project key, and issue type for alert dispatching.
+        These values should be masked during input and are credential requirements.
+
+        An additional parameter 'aggregate' is used to determine if alerts are aggregated into a
+        single Jira issue based on the StreamAlert rule.
+
+        Returns:
+            OrderedDict: Contains various OutputProperty items
+        """
+        return OrderedDict([
+            ('descriptor',
+             OutputProperty(description='a short and unique descriptor for this '
+                                        'Jira integration')),
+            ('username',
+             OutputProperty(description='the Jira username',
+                            mask_input=True,
+                            cred_requirement=True)),
+            ('password',
+             OutputProperty(description='the Jira password',
+                            mask_input=True,
+                            cred_requirement=True)),
+            ('url',
+             OutputProperty(description='the Jira url',
+                            mask_input=True,
+                            cred_requirement=True)),
+            ('project_key',
+             OutputProperty(description='the Jira project key',
+                            mask_input=False,
+                            cred_requirement=True)),
+            ('issue_type',
+             OutputProperty(description='the Jira issue type',
+                            mask_input=False,
+                            cred_requirement=True)),
+            ('aggregate',
+             OutputProperty(description='the Jira aggregation behavior to aggregate '
+                                        'alerts by rule name (yes/no)',
+                            mask_input=False,
+                            cred_requirement=True))
+        ])
+
+    def _search_jira(self, jql, max_results=100, validate_query=True, fields=None):
+        """Search Jira for issues using a JQL query
+
+        Args:
+            jql (str): The JQL query
+            max_results (int): Maximum number of results to return
+            validate_query (bool): Whether to validate the JQL query or not
+            fields (list): List of fields to return for each issue
+        Returns:
+            list: list of issues matching JQL query
+        """
+        search_url = os.path.join(self.base_url, self.SEARCH)
+        headers = {'Content-Type': 'application/json', 'cookie' : self.auth_cookie}
```

Comment:
Remove the extra space after `'cookie'` here. I noticed a few other instances like this in the file as well. The following command is what we've been using to help with small things like this:

`yapf --in-place /path/to/file.py`

Reply:
After running this on outputs.py, it looks like there are a lot of changes made to almost every output.  Do you want me to add these changes?